### PR TITLE
Us7.1 timeline accessibility

### PR DESCRIPTION
**Pull request by ktpa**
Trying this again, but closing 7 is kind of harsh. This is part of 7 and 7 will be reopened upon merge.

### CODE REVIEW

File: frontend/src/components/G6Graph.svelte [hunk 1]
```diff
@@ -115,6 +117,19 @@
 
         graph.on("nodeselectchange", (e) => dispatch("nodeselected", e));
 
+        // Enable keyboard manipulation
+        graph.on("keydown", (e: IG6GraphEvent) => {
+            if (e.key === "ArrowRight") {
+                graph.translate(-50, 0);
```
Comment (RuthgerD) [inline frontend/src/components/G6Graph.svelte:123]: 50 is a constant that should be a variable somewhere instead of being hardcoded in this place.

Comment (ktpa) [reply frontend/src/components/G6Graph.svelte]: Thank you. Should be solved as of commit c12343f.

File: frontend/src/components/G6Graph.svelte [hunk 2]
```diff
@@ -21,13 +23,15 @@
         dispatch("nodeselected", null);
     };
 
+    // This is a hack to get the graph to render properly
+    // and not be cut off prematurely in width
     export const resizeGraph = () => {
         if (graph && container) {
             const width = Number(
-                window.getComputedStyle(container).width.replace("px", "")
+                window.innerWidth
```
Comment (kam56) [inline frontend/src/components/G6Graph.svelte:31]: Nice and simple solution. Great job!

**Review by kam56 — COMMENTED** [review]
Nice work overall. I run it locally. Everything works as expected.

**Review by kam56 — APPROVED** [review]
I run it locally, so it works as expected. Nice work!

**Pull request merged**